src/app/components/case/Workflow/Workflow.tsx

```diff
@@ -55,7 +61,7 @@ const Workflow: React.FC<Props> = ({ id }) => {
         ))
       ) : (
         <>
-          <>Geen taken beschikbaar. </>
+          <>{isClosed ? "Deze zaak is afgesloten, er zijn op dit moment geen open taken." : "Geen taken beschikbaar."} </>
```

Comment:
Added a bit more descriptive message for when a case is closed. The fallback should never be shown since we're either 'busy' or 'polling' (or there are are open tasks), but good to have a fallback obviously...